src/app/shared/guards/go-home.guard.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree, Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AuthService } from '../services/auth.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GoHomeGuard implements CanActivate {
```

Comment:
GoHome make the sense that it redirect me to home, I'm not so sure that is the right name